lib/prependers.rb

```diff
@@ -12,5 +12,21 @@ def load_paths(*paths, **options)
         Loader.new(path, options).load
       end
     end
+
+    def setup_for_rails(load_options = {})
+      prependers_root = Rails.root.join('app', 'prependers')
+
+      Rails.autoloaders.main.ignore(prependers_root) if Rails.autoloaders.zeitwerk_enabled?
```

Comment:
As far as I got, there's no reason to have this into the `to_prepare` block. This will run at every code reload while we can just set it up before Zeitwerk is run the first time at boot.

Reply:
@kennyadsl outside `to_prepare`, `Rails.autoloaders.zeitwerk_enabled?` seems to be `false` even in apps that use Zeitwerk. I guess the autoloader hasn't been initialized at that point yet.